### PR TITLE
[DTGB-461] Titel varianten

### PR DESCRIPTION
**Pull request by Jeroen005**
# Ticket
Link to ticket: https://digipolisgent.atlassian.net/browse/DTGB-461

### CODE REVIEW

**Review by meirege — APPROVED** [review]

File: styleguide/components/31-molecules/titles/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Titles
```
Comment (delrueba) [inline styleguide/components/31-molecules/titles/README.md]: :medal_sports:

File: styleguide/components/31-molecules/titles/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+There a few title variants available in style guide.
+
+- **Overview titles:** Can be used above an overview, mostly as the page title.
+  This variant is always styled as a heading 1.
+- **Collection titles:** Can be used above an collection, mostly as a the of a
+  block or region. This variant is always styled as a heading 2.
```
Comment (delrueba) [inline styleguide/components/31-molecules/titles/README.md]: Something's not right about this sentence. 
Would propose the more generic 'above a collection of teasers or tiles'.

File: styleguide/components/31-molecules/titles/titles.config.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+  title: 'Titles',
```
Comment (delrueba) [inline styleguide/components/31-molecules/titles/titles.config.js]: Single not plural, you only show one 'title'.
See 'button', 'link'...

**Review by delrueba — CHANGES_REQUESTED** [review]
Overall I feel it's confusing to have 'headings' in atoms and 'titles' in molecules.

I know... technically... these titles consist of two atoms and are in fact molecules.
But if I would use styleguide as an external developer and want to use a title: I'd expect all variants to be together.

We could use this setup with the 'type' context to putt all headings in a single preview and consistently use '@title' everywhere we need any type of title.

**Review by delrueba — APPROVED** [review]

**Pull request merged**